pyproject.toml

```
@@ -114,6 +116,7 @@ docs = [
     "towncrier>=22.8.0",
     "scipy>=1.7.0", # examples using scipy
     "slack-sdk>=3.4.2", # slack example notebook
+    "qcodes_loop>=0.1.1", # leacy import examples
```

to disambiguate import of legacy dataset from import of python packages
```suggestion
    "qcodes_loop>=0.1.1", # legacy dataset import examples
```